src/components/PoolingWidget/DefineSpread.tsx

```diff
@@ -35,17 +40,53 @@ export const SpreadInformation: React.FC<SpreadInformationProps> = ({ selectedTo
   )
 }
 
-const DefineSpread: React.FC<DefineSpreadProps> = ({ selectedTokensMap, spread, setSpread }) => {
-  const handleSpreadChange = ({ target: { value } }: React.ChangeEvent<HTMLInputElement>): void => {
-    if (+value < 0 || +value >= 100) return
+const DefineSpread: React.FC<DefineSpreadProps> = ({ selectedTokensMap, spread, setSpread, nextStep }) => {
+  const onChange = useCallback(
+    ({ currentTarget: { value } }: React.SyntheticEvent<HTMLInputElement>): void => {
+      if (+value === NaN || +value <= 0 || +value >= 100) {
```

Comment:
use `isNaN`

Reply:
second that, nobody equals NaN, even `NaN === NaN -> false`